src/Components/Header/Header.css

```diff
@@ -0,0 +1,38 @@
+.header-container {
+  margin-top: 43px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.header-container ul {
+  list-style-type: none;
+  display: flex;
+  -moz-column-gap: 35px;
+       column-gap: 35px;
+  align-items: center;
+}
+.header-container ul a {
```

Comment:
- [ ] I can see few occurrences here with the elements using styles without any classnames. Apply classnames for these.